mrp_bom_current_stock/readme/DESCRIPTION.md

```
@@ -0,0 +1,3 @@
+This modules extend the Manufacturing App adding a report that explodes
+the bill of materials and show the stock available in the source
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)